io/zenoh-transport/src/unicast/establishment/ext/patch.rs

```diff
@@ -0,0 +1,196 @@
+    type RecvInitAckOut = ();
+    async fn recv_init_ack(
+        self,
+        input: Self::RecvInitAckIn,
+    ) -> Result<Self::RecvInitAckOut, Self::Error> {
+        let (state, other_ext) = input;
+        state.patch = other_ext;
+        Ok(())
+    }
+
+    type SendOpenSynIn = &'a StateOpen;
+    type SendOpenSynOut = ();
+    async fn send_open_syn(
+        self,
+        _state: Self::SendOpenSynIn,
+    ) -> Result<Self::SendOpenSynOut, Self::Error> {
+        unimplemented!()
+    }
+
+    type RecvOpenAckIn = (&'a mut StateOpen, ());
+    type RecvOpenAckOut = ();
+    async fn recv_open_ack(
+        self,
+        _state: Self::RecvOpenAckIn,
+    ) -> Result<Self::RecvOpenAckOut, Self::Error> {
+        unimplemented!()
+    }
+}
+
+/*************************************/
+/*            ACCEPT                 */
+/*************************************/
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub(crate) struct StateAccept {
+    patch: init::ext::PatchType,
+}
+
+impl StateAccept {
+    pub(crate) const fn new() -> Self {
+        Self {
+            patch: init::ext::PatchType::NONE,
+        }
+    }
+
+    pub(crate) const fn get(&self) -> init::ext::PatchType {
+        self.patch
+    }
+
+    #[cfg(test)]
+    pub(crate) fn rand() -> Self {
+        Self {
+            patch: init::ext::PatchType::rand(),
+        }
+    }
+}
+
+// Codec
+impl<W> WCodec<&StateAccept, &mut W> for Zenoh080
+where
+    W: Writer,
+{
+    type Output = Result<(), DidntWrite>;
+
+    fn write(self, writer: &mut W, x: &StateAccept) -> Self::Output {
+        let raw = x.patch.raw();
+        self.write(&mut *writer, raw)?;
+        Ok(())
+    }
+}
+
+impl<R> RCodec<StateAccept, &mut R> for Zenoh080
+where
+    R: Reader,
+{
+    type Error = DidntRead;
+
+    fn read(self, reader: &mut R) -> Result<StateAccept, Self::Error> {
+        let raw: u8 = self.read(&mut *reader)?;
+        let patch = init::ext::PatchType::new(raw);
+        Ok(StateAccept { patch })
+    }
+}
+
+#[async_trait]
+impl<'a> AcceptFsm for &'a PatchFsm<'a> {
+    type Error = ZError;
+
+    type RecvInitSynIn = (&'a mut StateAccept, init::ext::PatchType);
+    type RecvInitSynOut = ();
+    async fn recv_init_syn(
+        self,
+        input: Self::RecvInitSynIn,
+    ) -> Result<Self::RecvInitSynOut, Self::Error> {
+        let (state, other_ext) = input;
+        state.patch = other_ext;
+        Ok(())
+    }
+
+    type SendInitAckIn = &'a StateAccept;
+    type SendInitAckOut = init::ext::PatchType;
+    async fn send_init_ack(
+        self,
+        _state: Self::SendInitAckIn,
+    ) -> Result<Self::SendInitAckOut, Self::Error> {
+        Ok(init::ext::PatchType::CURRENT)
+    }
+
+    type RecvOpenSynIn = (&'a mut StateAccept, ());
+    type RecvOpenSynOut = ();
+    async fn recv_open_syn(
+        self,
+        _state: Self::RecvOpenSynIn,
+    ) -> Result<Self::RecvOpenSynOut, Self::Error> {
+        unimplemented!()
```

Comment:
Add a comment that this is not `unimplemented` but rather no extension exists for that. 
Or should we simply return `Ok`?